src/systems/userinput/userinput.js

```diff
@@ -223,6 +223,10 @@ AFRAME.registerSystem("userinput", {
         gamepadDevice = new OculusTouchControllerDevice(e.gamepad);
       } else if (e.gamepad.id === "Oculus Go Controller") {
         gamepadDevice = new OculusGoControllerDevice(e.gamepad);
+
+        // For Oculus Go, to increase perf, disable all non-essential devices
+        // TODO detect if keyboard/mouse are available
+        this.activeDevices = new Set([new HudDevice()]);
```

Comment:
I would just change the initial declaration above to initialize a different set of devices based on platform. Doing it on controller connection is a bit scary. Also having the keybaord device is helpful for hitting "l" in the debugger to get frame output, but we can probably just add a console command for that.

Pretty surprised these are hotspots, they are doing very little work, but good to know.